packages/pdf-viewer/Page.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useRef, useState } from 'react';
+import useIsVisible from './hooks/useIsVisible';
+import { PdfData, ScaledSize } from './pdfSource';
+
+require('./pages.css');
+
+export default function Page(props: {
```

Comment:
Defined the props in a new Props interface. No inline definitions anywhere please.